src/app/model/server/incident-update.ts

```diff
@@ -21,7 +21,7 @@ export function loadIncidentUpdates(config: AppConfigService, id: string, http:
                 return response.data;
             }),
         );
-    } catch (e: any) {
+    } catch (e: unknown) {
```

Comment:
Errors can usually be left without a type annotation.

Its good practice to cast them to [`HttpErrorResponse`](https://angular.io/api/common/http/HttpErrorResponse) inside the `catch` if further error handling is required.

Reply:
I'll make sure to take note of this.